### PR TITLE
[PoC] [WiP] Find cards by base card

### PR DESCRIPTION
**Pull request by Dajakk**
Legendary cards and farms through the seasons cards should behave as their basic counterparts for most of the effects

### CODE REVIEW

**Comment by tadeaustria** [comment]
I cleaned up the old function, but I think this is good

**Comment by tadeaustria** [comment]
OK, I have to revert my statement. The old function is still needed for the wife points. Maybe we can get rid of it, with #45

**Review by tadeaustria — APPROVED** [review]

**Comment by Dajakk** [comment]
> The old function is still needed for the wife points. Maybe we can get rid of it, with #45

I had a feeling it might not be as simple as replacing one with the other for now, but had no computing power to analyze each place. Step by step ;]

**Comment by Dajakk** [comment]
@tadeaustria Please avoid force pushes. Now I have conflicts when trying to pull your changes

File: src/player.js [hunk 1]
```diff
@@ -167,8 +167,12 @@ class Player {
         return this.findCountFct((card) => { return card.name == cardToFind.name; });
     }
 
+    findCountBaseCard(cardToFind) {
+        return this.findCountFct((card) => { return card.name == cardToFind.name || card.baseCardName === cardToFind.name; });
+    }
+
     findCountHusbandMatches() {
-        return this.getWifeCount4AdditionalPoints() + this.findCountCard(basecards['mayberrymatriarch']);
+        return this.getWifeCount4AdditionalPoints();
```
Comment (Dajakk) [inline src/player.js]: @tadeaustria Shouldn't this be left unchanged?

Comment (Dajakk) [reply src/player.js]: Other than that I think everything is ok

Comment (tadeaustria) [reply src/player.js]: Yes, thanks you are right. Without it, the Mayberry didn't take up a space if placed alone.

**Comment by tadeaustria** [comment]
> @tadeaustria Please avoid force pushes. Now I have conflicts when trying to pull your changes

Sorry, but that I cannot guarantee. In the end I wanted to have one nice clean commit, for this issue. But I agree in this case the cleanup could have been postponed.
To get to the latest branch use simply `git fetch` + `git reset --hard {remote}/{branch}` to avoid merge conflicts (asuming you do not have unpushed local changes). But @Dajakk please do not push, if you did not change anything again...

**Comment by Dajakk** [comment]
> In the end I wanted to have one nice clean commit, for this issue. But I agree in this case the cleanup could have been postponed.

Isn't this what 'Squash and merge' button is for? I'm no expert to GitHub, I'm working with BitBucket, but the description suggests it is exactly for that purpose.

Sorry about the push, I wanted to clean this up and added very small change that got mixed up somewhere (missing semicolon)

**Comment by tadeaustria** [comment]
basically yes - as I've said, in this case it would have worked

**Comment by Dajakk** [comment]
Btw, maybe we could establish any other means of communications? Those comments are at least suboptimal in my opinion

**Comment by tadeaustria** [comment]
What type of communication are you thinking about?

**Comment by Dajakk** [comment]
Any IM? Right now I'm more or less active on Messenger and WhatsApp, but we could agree on anything

**Pull request merged**